Documentation/Docker-setup/jobmanager-entrypoint.sh

```
@@ -0,0 +1,170 @@
+            \"programArgs\": \"--config.content $CONFIG_B64\"
+        }"
+
+    echo "Job submitted."
+}
+
+check_api_job_status() {
+    local JOB_NAME=$1
+    local API_URL=${HEALTH_API_URL}
+    local TOKEN=${AUTH_TOKEN}
+
+    local RESPONSE=$(curl -s --location "$API_URL" --header "Authorization: $TOKEN")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)